httpapi/httpapi.go

```diff
@@ -23,16 +41,39 @@ func NewServeMux(nodes []*node.HydraNode) *http.ServeMux {
 				Addrs: n.Host.Addrs(),
 			})
 		}
-	})
+	}
+}
 
-	return mux
+// "/records/fetch" Receive a record and fetch it from the network, if available
```

Comment:
🙏 please can we have docs for these new endpoints in the README?

Reply:
Of course! :)